README.md

```diff
@@ -123,6 +123,8 @@ window.setTimeout(function () {
 }, splashDuration - fadeDuration);
 ```
 
+### iOS Quirks
```

Comment:
@dhams, looks like you have added a few extra lines - compare with the current version in master: https://github.com/apache/cordova-plugin-splashscreen/blame/master/README.md#L94